conf/nfcore_license.config

```diff
@@ -0,0 +1,13 @@
+//Profile config names for nf-core/configs
+params {
+    config_profile_description = 'Sentieon license set up for nf-core by nf-core/configs'
+    config_profile_contact = 'Maxime U Garcia (@maxulysse)'
+    config_profile_url = 'https://www.sentieon.com'
+}
+
+process {
+    withLabel: 'sentieon' {
+        ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64 ?: ''
```

Comment:
`secrets.SENTIEON_AUTH_MECH_BASE64 ?: ''` failed for Ram when `secrets.SENTIEON_AUTH_MECH_BASE64` wasn't set.

Reply:
So the idea is to have many different licenses in `conf/nfcore_license.config`? Say `nfcore_license.config` refers to nf secrets for tool X and tool Y but the user only has license for X and can only set the nf secrets for X, then it seems to me that this will fail for the same reasons that `test_cache` is failing when the user doesn't have a test-license for Sentieon.

Reply:
yeah, this should only works when we set up secrets when we start the tests, similarly to how we do it within our CI